registry/mongo/container.yaml

```diff
@@ -15,8 +15,13 @@ tags:
   5.0.3: sha256:4088649f737cf704deaf350ccd5ad8045552c5a0f8a5a2e81c1c23e280db2d80
   5.0.4: sha256:cf9f5df5419319390cc3b5d9abfc2d0d0b149b3e04a6c9936990129e3e29b579
   5.0.5: sha256:079089900e9511a782a59a4276046835189720eb668088869d147d1145cebe14
-  5.0.6: sha256:c7f5b78eead63e672176d4a6e0932f8649c8f727dec5acef6b9ed7c9bdc3dae5
-  latest: sha256:c7f5b78eead63e672176d4a6e0932f8649c8f727dec5acef6b9ed7c9bdc3dae5
+  5.0.6: sha256:fed6248ae0bb0d54c0448eb786c87120737eedc522172ee1536ad47789782348
+  latest: sha256:fed6248ae0bb0d54c0448eb786c87120737eedc522172ee1536ad47789782348
+  '5': sha256:fed6248ae0bb0d54c0448eb786c87120737eedc522172ee1536ad47789782348
+  5-nanoserver-ltsc2022: sha256:28b9079c157d0f2c23fec6352a37d85e2c20947c721214024b3aea6338341ff7
+  5-windowsservercore-ltsc2016: sha256:9620d60541d45e913c30f78623ee4b5ec8f650399339982dda11e9cd1d1b9c78
+  5-nanoserver-1809: sha256:89c4b070fbc207f5b28f69cff5dec9b841ace73c3c7baeb09b598c060f173699
+  '5.0': sha256:fed6248ae0bb0d54c0448eb786c87120737eedc522172ee1536ad47789782348
 aliases:
```

Comment:
Why is it inserting the flavours of `5` but not the flavours of `5.0.6` ? Is it because `nanoserver` etc end up like minor version number ?

Reply:
We only consider the major and minor (e.g., 5.0) so if the parser has already seen 5.0 (without a third) it will not include 5.0.6. The idea behind that is that 5.0.6 might be deprecated soon, but the 5.0 will be a "last update of 5.0 with all changes for that version family). This also assumes that the person is not saying that 5.0 == 5.0.0 - if they used 5.0.0 it would be ordered as less than 5.0.6, and that would be chosen first.